src/snowcli/cli/stage/manager.py

```diff
@@ -40,11 +41,42 @@ def put(
             f"auto_compress=false parallel={parallel} overwrite={overwrite}"
         )
 
+    def _put(
+        self,
+        local_path: Union[str, Path],
+        stage_path: str,
+        role: Optional[str] = None,
+        parallel: int = 4,
+        overwrite: bool = False,
+    ) -> SnowflakeCursor:
+        """
+        Internal only method, created for Native Apps use case.
```

Comment:
Would suggest documenting what the function does, rather than why it was created.